lib/oidc/handleOAuthResponse.ts

```diff
@@ -32,8 +33,8 @@ import { verifyToken } from './verifyToken';
 import { getDefaultTokenParams } from './util';
 
 function validateResponse(res: OAuthResponse, oauthParams: TokenParams) {
-  if (res['error'] || res['error_description']) {
-    throw new OAuthError(res['error'], res['error_description']);
+  if (res['error'] || res['error_description']) { // TODO: should this be an AND instead of an OR condition?
```

Comment:
if either `'error'` nor `'error_description'` can be null, the `if` condition feels like it should be an `AND`

Reply:
YES. I completely agree. I agonized over this because I didn't want to create a breaking change, but now that it is in 6.x it is the perfect chance to fix this.  Good catch!